src/Components/Article/CommentList.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+function CommentList() {
+  return <div></div>;
+}
+
```

Comment:
Think I've typed this before too but this component is not related to this PR so shouldn't be included, please.